wazuhapp/generate_wazuh_app.sh

```diff
@@ -65,7 +66,7 @@ delete_sources(){
 }
 
 main(){
-
+    cheksum="no"
```

Comment:
Typo => `cheksum`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/9d09627160a93091dae44ea05cd9ca3429cd8f21